index.js

```diff
@@ -351,21 +351,20 @@ function onReady () {
             break;
 
             // Global Shortcut
-            case consts.eventNames.globalShortcutCmdRegister:
-            const isRegistered = globalShortcut.register(json.globalShortcut.accelerator, () => {
-                client.write(json.targetID, consts.eventNames.globalShortcutEventTriggered, {GlobalShortcut:{accelerator: json.globalShortcut.accelerator}});
+            case consts.eventNames.globalShortcutEventRegistered:
+            const isRegistered = globalShortcut.register(json.globalShortcuts.accelerator, () => {
+                client.write(json.targetID, consts.eventNames.globalShortcutEventTriggered, {GlobalShortcuts:{accelerator: json.globalShortcuts.accelerator}});
```

Comment:
You need to use `globalShortcuts` with a lower `g` instead of `GlobalShortcuts` in the json. Same thing below